content/posts/2019/05/sprintseoul.eng.md

```diff
@@ -0,0 +1,90 @@
+---
+title: Libplanet Team will participate in Sprint Seoul
+date: 2019-05-20
+authors: [hong.minhee]
+translators: [kidon.seo]
+---
+
+Did you know [Sprint Seoul][1] will be held in Gangnam this June 29th (Sat)? 
+Sprint Seoul is an open source event that gathers people passionate about open source culture and encourage them to contribute codes and/or documents to open source projects. 
+Even those who are interested in open source but have not been able to contribute can also come to the event and experience their first open source contribution.
+
+> Sprint invites both authors and contributors of open source projects to get together and find/resolve issues in a short period of time,
+> enabling both parties to dive deep into the projects.
+
+Our team has already participated in the event last April, and we’re continuing our participation this coming June. We are participating as the leader of [Libplanet] Project, and we look forward to meeting you all at the event to further develop our project.
+
+Applications for Sprint Seoul will only be accepted until June 20th, and details of the [application form][2] and the event can be found on the [official website][1].
```

Comment:
This `[2]` overlaps with `[2]: {{< ref "libplanet-0.2.eng.md" >}}` below.
How about changing it to `[application form]`?
```suggestion
Applications for Sprint Seoul will only be accepted until June 20th, and details of the [application form] and the event can be found on the [official website][1].
```